### PR TITLE
#154 : Add homepage to Cargo.toml

### PR DESCRIPTION
**Pull request by hiirrxnn**
Also includes new changes requested by collaborators ! Let me know if there are changes to be made !

### CODE REVIEW

**Comment by Fokko** [comment]
@hiirrxnn oops, I just merged another PR, and now have merge conflicts 🤦

**Comment by hiirrxnn** [comment]
Is there something that needs to be done from my side?

File: crates/test_utils/Cargo.toml [hunk 1]
```diff
@@ -17,8 +17,9 @@
 
 [package]
 name = "iceberg_test_utils"
-version = "0.1.0"
-edition = "2021"
+version = {workspace = true} 
+edition = {workspace = true} 
+homepage = {workspace = true}
```
Comment (liurenjie1024) [inline crates/test_utils/Cargo.toml:23]: I think we also need repository and license for this crate?

**Comment by liurenjie1024** [comment]
@hiirrxnn Thanks for the contribution! 
A small tip: you can run `make check` to verify things like formatting, linting.

**Comment by hiirrxnn** [comment]
Noted ! I'm new to github , will definitely keep this in mind for the future . I have closed previous PR as you suggested !

File: crates/test_utils/Cargo.toml (hunk 1 above)
Comment (hiirrxnn) [reply crates/test_utils/Cargo.toml:23]: done!

**Review by liurenjie1024 — APPROVED** [review]
LGTM, thanks!

**Comment by liurenjie1024** [comment]
cc @Fokko PTAL

**Comment by Fokko** [comment]
Looks like there are conflicts?

**Comment by liurenjie1024** [comment]
cc @hiirrxnn Could you rebase with main branch?

**Comment by hiirrxnn** [comment]
But the PR with main branch has been closed . Should i do it anyway?

**Comment by hiirrxnn** [comment]
Also , feature anyway has all the changes of main since it has been built on top of it , or should i rebase main with feature?

**Comment by liurenjie1024** [comment]
> Also , feature anyway has all the changes of main since it has been built on top of it , or should i rebase main with feature?

Hi, @hiirrxnn You can learn how to resolve conflicts here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

**Comment by hiirrxnn** [comment]
Should i rebase feature (the branch whose PR this is) with main? Or should i merge?

**Comment by liurenjie1024** [comment]
> Should i rebase feature (the branch whose PR this is) with main? Or should i merge?

I think both are ok to me.

**Comment by hiirrxnn** [comment]
Please review .

**Review by liurenjie1024 — APPROVED** [review]
Thanks!

**Review by Fokko — APPROVED** [review]

**Comment by liurenjie1024** [comment]
Hi, @hiirrxnn The `cargo sort` check failed, you can run `cargo sort -w` to fix it.

**Comment by hiirrxnn** [comment]
Please review !

**Review by liurenjie1024 — APPROVED** [review]
LGTM, Thanks!

**Comment by Fokko** [comment]
Thanks @hiirrxnn for working on this 🙏

**Comment by hiirrxnn** [comment]
I'm glad to have collaborated with you guys!

**Pull request merged**